### ISSUE TITLE
[BUG] Entries with over 2000 characters are not saved, no error message

### ISSUE THREAD

**Issue by mark-hellmonds** [reliatec-gmbh/LibreClinica#391]
**Description:**
Text and textarea items are technically limited to 3999 characters. However, it is not possible to store more than 2000 characters in a single item. Instead of throwing an error, the CRF saves as normal, simply dropping the long entry.
After some investigation, the cause seems to be the audit logs, as they are limited to 2000 characters, and changing them to a higher value (e.g. 4000) allows long entries to be saved.
However, the reason for the low audit log character limit is unknown, as is the effect of changing the limit back to a higher number.

**Requirements:**
Scheduled event with a CRF containing a text or textarea item.

**Steps to follow:**
1. login
2. open the CRF in DataEntry mode
3. enter more than 2000 and less than 3999 characters in the text/textarea item
4. click save

**Expected result:**
All data including the over 2000 characters are saved.

**Actual result:**
The CRF is saved without any error message, but when the CRF is viewed again, the text/textarea item is empty.

**Server Setup (optional):**
 - OS [Windows]
 - LibreClinica 1.2.1

**Comment by toskrip** [comment]
you are right, I think we can increase a the limit for old_value and new_value in audit_log_event table without any side effects. I will try to make DB changeset for 1.3 release targeting this.
